lahja/base.py

```diff
@@ -219,6 +219,12 @@ class BaseEndpoint(EndpointAPI):
 
     logger = logging.getLogger("lahja.endpoint.Endpoint")
 
+    def __str__(self) -> str:
+        return f"Endpoint[{self.name}]"
+
+    def __repr__(self) -> str:
+        return f"<{self.name}>"
```

Comment:
nitpick: I think these two are flipped, [`__repr__`](https://docs.python.org/3/reference/datamodel.html#object.__repr__) is the one which roughly looks like a constructor. What's the rationale for implementing both? I think `__str__` is usually added if `__repr__` gives output which is too long, but here they both give short and printable output.